tests/test_quoteengine.py

```diff
@@ -0,0 +1,124 @@
+from datetime import datetime
+from unittest.mock import patch
+from unittest.mock import MagicMock
+
+import pytest
+
+from tululbot.utils import QuoteEngine
+
+
+class FakeResponse:
+    def get_json(self):
+        return self.json
+
+    pass
+
+
+@pytest.fixture
+def qe():

```

Comment:
can you please use a more descriptive name? it's not obvious what `qe` stands for.